frontend/package.json

```diff
@@ -12,7 +12,13 @@
 		"lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore"
 	},
 	"dependencies": {
-		"vue": "^3.2.38"
+		"@fortawesome/fontawesome-svg-core": "^6.2.0",
```

Comment:
What are all these ? Isn't it a lot for just font awesome ? I think we can do better (i mean smaller) but it's not a priority !!